### DISCUSSION TITLE
Differences between JVM and Android

### DISCUSSION THREAD

**Discussion by Doomsdayrs** [korlibs/korge-luak#8] [Q&A] [answered]
Aside from JVM using java 1.8 and Android using java 11, what other differences are there?

**Comment by soywiz** [comment, marked as answer]
The JVM exposes it as a `javax.script`. Other than that I believe they are the same IIRC. Looking at the code I cannot see further differences.

**Reply by Doomsdayrs** [reply]
And question for sanity, does the android variant inherit its code from the jvm variant?

**Reply by soywiz** [reply]
I think they share the same sourceSet for the common code, and then there is a separate sourceset for the JVM.
Why do you ask? Is there something do you believe is wrong somewhere?
